conf/test_nothing.config

```
@@ -0,0 +1,46 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/eager -profile test,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name        = 'Test profile'
+    config_profile_description = 'Minimal test dataset to check pipeline function'
+
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus   = 2
+    max_memory = '6.GB'
+    max_time   = '6.h'
+
+    // Input data
+    // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
+    // TODO nf-core: Give any required params for the test so that command line flags are not needed
+    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/eager/testdata/Human/human_design_bam_eager3.tsv'
+
+    // Genome references
+    fasta = 'https://raw.githubusercontent.com/nf-core/test-datasets/eager/reference/Human/hs37d5_chr21-MT.fa.gz'
+
+    skip_preprocessing = true
+    skip_deduplication = true
+    skip_qualimap      = true
+    skip_damage_calculation = false
+    mapstats_skip_preseq = false
```

```suggestion
    mapstats_skip_preseq = true
```

But confused why they are turned on, also shouldn't that parameter be flipped (skip first/) ? Something to address later Ig uess

---

You made this new profile, but I agree that it should be turned off for now.